docs/advanced-features/middleware.md

```diff
@@ -98,7 +98,7 @@ export const config = {
      * - static (static files)
      * - favicon.ico (favicon file)
      */
-    '/((?!api|static|favicon.ico).*)',
+    '/((?!api|_next/static|favicon.ico).*)',
```

Comment:
This is going against the comment so we should probably update that to specifically say `_next/static`

Reply:
Done!